corehq/apps/reports/models.py

```diff
@@ -760,6 +760,9 @@ def send(self):
             self.delete()
             return
 
+        if self.owner.is_deleted():
+            return
```

Comment:
at first glance seems like that will just be one less saved report to try every time